Apps/WebClient/src/Server/Controllers/RobotsController.cs

```diff
@@ -58,5 +60,20 @@ public ActionResult Robots()
             };
             return result;
         }
+
+        [SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Team decision")]
+        private static string? ReadRobotsFile(string robotsFilePath)
+        {
+            string? defaultRobotsAssetContent = AssetReader.Read("HealthGateway.WebClient.Server.Assets.Robots.txt");
+
+            try
+            {
+                return !string.IsNullOrEmpty(robotsFilePath) ? System.IO.File.ReadAllText(robotsFilePath) : defaultRobotsAssetContent;
```

Comment:
Nit: using statements are preferred in place of fully-qualified names.  System.IO.File.ReadAllText() -> File.ReadAllText()